app/Jobs/ImportPipeline/ImportInstagram.php

```diff
@@ -101,7 +101,11 @@ public function handle()
 
                 $path = storage_path("app/$importData->path");
                 $storagePath = "public/m/{$monthHash}/{$userHash}";
-                $newPath = "app/$storagePath/$filename";
+                $dir = "app/$storagePath";
+                if(!is_dir(storage_path($dir))) {
+                    mkdir(storage_path($dir), 0777, true);
```

Comment:
Please remove the second and third parameters of the `mkdir` function

Reply:
I think the third parameter is necessary in order to correctly create the parent directory recursively.

Reply:
I tried removing the second and parameter, but the import did not work afterwards. The second parameter could be omitted, PHP 8 named arguments make it possible:

```php
mkdir(storage_path($dir), recursive: true);
```

However, since PHP versions <8.0 are also targeted, I would leave it just as is.